components/Forms/CreateMemoryForm.js

```diff
@@ -1,57 +1,133 @@
+import { useState } from "react";
 import styled from "styled-components";
-import BookmarkButton from "../BookmarkButton";
 import { StyledButtonFrame } from "../GlobalStyles";
 import InputAndLabelTextPair from "./Input/InputAndLabelTextPair";
 
-export default function CreateMemoryForm({ onAddCreateCard }) {
+import InputAndLabelDateCheck from "./Input/InputAndLabelDateCheck";
+
+export default function CreateMemoryForm({ onAddCreateCard, onImage }) {
+  const [counter, setCounter] = useState();
+
+  const maxLengthTextarea = 200;
+
+  function handleCounter(event) {
+    setCounter(event.target.value.length);
+  }
+
   return (
     <StyledCreateForm aria-label="create card form" onSubmit={onAddCreateCard}>
```

Comment:
`<StyledCreateForm aria-label="create card form" onSubmit={onAddCreateCard;setCounter(0)}>
`
does this work?

Reply:
Great job, Marvin! 👏 
```js
onSubmit={()=>{onAddCreateCard; setCounter(0)}}
```

Reply:
Hi Andy, are you sure that your event is passed with the `onAddCreateCard`? I think it would be better to write this explicitly in a handleSubmit function.